KizGetdown.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- Footer -->
+
+    <link
+      href="/Users/luba/Desktop/WEB-DEVELOPMENT/STYLING/fontawesome-free-5.13.0-web/css/all.css"
```

Comment:
This link only works on your computer. You must never use absolute local paths (e.g. /Users/luba...). Use relative paths (e.g. your CSS) or online absolute paths (e.g. bootstrap link).

You need to add this font awesome file to the repository and change the href to a relative path. A common folder structure is:

```bash
index.html
assets/
├─ images/ # your images
├─ styles/ # your css
├─ scripts/ # your javascript
├─ vendor/ # external code. e.g fontawesome